### PR TITLE
rinkarby deprecation; remove all rinkarby stuff + fix broken links

### PR DESCRIPTION
**Pull request by DZGoldman**

### CODE REVIEW

**Comment by DZGoldman** [comment]
(this now includes https://github.com/OffchainLabs/arbitrum-docs/pull/53 just to avoid conflicts etc.)

**Review by fionnachan — COMMENTED** [review]
do we need to remove the rinkeby examples on L11 and L43 of `state-migration.mdx`?

**Comment by DZGoldman** [comment]
> do we need to remove the rinkeby examples on L11 and L43 of state-migration.mdx?
both examples @fionnachan mentioned reference the "classic instance directory - e.g. `${HOME}/.arbitrum/rinkeby/nitroexport`.

What's the equivalent directory for mainnet/ArbOne? @joshuacolvin0 @Tristan-Wilson

**Comment by DZGoldman** [comment]
@fionnachan updated

File: arbitrum-docs/migration/state-migration.mdx [hunk 1]
```diff
@@ -8,7 +8,7 @@ The recommended way to initialize a nitro node is to point it to a pre-initializ
 
 ## Exporting Data from a Classic Arbitrum Node
```
Comment (joshuacolvin0) [inline arbitrum-docs/migration/state-migration.mdx:10]: `${HOME}/.arbitrum/mainnet/nitroexport`

File: arbitrum-docs/migration/state-migration.mdx [hunk 2]
```diff
@@ -1,14 +1,14 @@
...
 
 When importing history - Nitro's genesis block will be the block that follows the imported history, and not block 0.
 
 The recommended way to initialize a nitro node is to point it to a pre-initialized database using the `--init.url` option. This documentation is for users who wish to create the full state on their own classic node.
 
 ## Exporting Data from a Classic Arbitrum Node
 
-Launch node with the option `--node.rpc.nitroexport.enable=true` note: this is only recommended for nodes with no public/external interfaces. All exported data will be written to directory "nitroexport" under the classic instance directory - e.g. `${HOME}/.arbitrum/rinkeby/nitroexport`.
+Launch node with the option `--node.rpc.nitroexport.enable=true` note: this is only recommended for nodes with no public/external interfaces. All exported data will be written to directory "nitroexport" under the classic instance directory - e.g. `${HOME}/.arbitrum/arb1/nitroexport`.
```
Comment (DZGoldman) [inline arbitrum-docs/migration/state-migration.mdx:11]:
```suggestion
Launch node with the option `--node.rpc.nitroexport.enable=true` note: this is only recommended for nodes with no public/external interfaces. All exported data will be written to directory "nitroexport" under the classic instance directory - e.g. `${HOME}/.arbitrum/mainnet/nitroexport`.
```

File: arbitrum-docs/useful-addresses.mdx [hunk 3]
```diff
@@ -33,30 +33,30 @@ _Users should only interact with the token bridge via dapp interfaces like https
...
+| L1 Weth               | [0xc02aaa39b223fe8d0a0e5c4f27ead9083c756cc2](https://etherscan.io/address/0xc02aaa39b223fe8d0a0e5c4f27ead9083c756cc2) | [0xC02aaA39b223FE8D0A0e5C4F27eAD9083C756Cc2](https://etherscan.io/address/0xC02aaA39b223FE8D0A0e5C4F27eAD9083C756Cc2)     | [0xB4FBF271143F4FBf7B91A5ded31805e42b2208d6](https://goerli.etherscan.io/address/0xB4FBF271143F4FBf7B91A5ded31805e42b2208d6)         |
+| L2 Weth               | [0x82aF49447D8a07e3bd95BD0d56f35241523fBab1](https://arbiscan.io/address/0x82aF49447D8a07e3bd95BD0d56f35241523fBab1)  | [0x722E8BdD2ce80A4422E880164f2079488e115365](https://nova.arbiscan.io/address/0x722E8BdD2ce80A4422E880164f2079488e115365) | [0xe39Ab88f8A4777030A534146A9Ca3B52bd5D43A3](https://goerli.arbiscan.arbitrum.io/address/0xe39Ab88f8A4777030A534146A9Ca3B52bd5D43A3) |
```
Comment (fionnachan) [inline arbitrum-docs/useful-addresses.mdx:47]: this is also a broken link
> https://goerli.arbiscan.arbitrum.io/address/0xe39Ab88f8A4777030A534146A9Ca3B52bd5D43A3

File: arbitrum-docs/useful-addresses.mdx [hunk 4]
```diff
@@ -79,8 +79,8 @@ These are the addresses of some other gateways that have been deployed and integ
...
+| ------------ | -------------------------------------------------------------------------------------------------------------------- | ------------------------------------------------------------------------------------------------------------------------- | ------------------------------------------------------------------------------------------------------------------------------------------- |
+| L2 Multicall | [0x7ecfbaa8742fdf5756dac92fbc8b90a19b8815bf](https://arbiscan.io/address/0x7ecfbaa8742fdf5756dac92fbc8b90a19b8815bf) | [0x5e1eE626420A354BbC9a95FeA1BAd4492e3bcB86](https://nova.arbiscan.io/address/0x5e1eE626420A354BbC9a95FeA1BAd4492e3bcB86) | [0x108B25170319f38DbED14cA9716C54E5D1FF4623](https://goerli-rollup-explorer.arbitrum.io/address/0x108B25170319f38DbED14cA9716C54E5D1FF4623) |
```
Comment (fionnachan) [inline arbitrum-docs/useful-addresses.mdx:84]: unrelated, should we change this from blockscout to arbiscan later?

File: arbitrum-docs/useful-addresses.mdx [hunk 5]
```diff
@@ -6,20 +6,20 @@ Here's some contract addresses that may be useful and/or of interest to those ex
...
+|                      | Mainnet: Arbitrum One                                                                                                 | Arbitrum Nova                                                                                                         | Nitro Goerli Rollup                                                                                                          |     |
+| -------------------- | --------------------------------------------------------------------------------------------------------------------- | --------------------------------------------------------------------------------------------------------------------- | ---------------------------------------------------------------------------------------------------------------------------- | --- |
```
Comment (fionnachan) [inline arbitrum-docs/useful-addresses.mdx:10]: can we remove the final empty column?

File: arbitrum-docs/useful-addresses.mdx [hunk 6]
```diff
@@ -33,30 +33,30 @@ _Users should only interact with the token bridge via dapp interfaces like https
...
+| L1 Weth               | [0xc02aaa39b223fe8d0a0e5c4f27ead9083c756cc2](https://etherscan.io/address/0xc02aaa39b223fe8d0a0e5c4f27ead9083c756cc2) | [0xC02aaA39b223FE8D0A0e5C4F27eAD9083C756Cc2](https://etherscan.io/address/0xC02aaA39b223FE8D0A0e5C4F27eAD9083C756Cc2)     | [0xB4FBF271143F4FBf7B91A5ded31805e42b2208d6](https://goerli.etherscan.io/address/0xB4FBF271143F4FBf7B91A5ded31805e42b2208d6)         |
+| L2 Weth               | [0x82aF49447D8a07e3bd95BD0d56f35241523fBab1](https://arbiscan.io/address/0x82aF49447D8a07e3bd95BD0d56f35241523fBab1)  | [0x722E8BdD2ce80A4422E880164f2079488e115365](https://nova.arbiscan.io/address/0x722E8BdD2ce80A4422E880164f2079488e115365) | [0xe39Ab88f8A4777030A534146A9Ca3B52bd5D43A3](https://goerli.arbiscan.arbitrum.io/address/0xe39Ab88f8A4777030A534146A9Ca3B52bd5D43A3) |
 
 </em>
 #### Third party gateways
```
Comment (fionnachan) [inline arbitrum-docs/useful-addresses.mdx:50]: need a line break before this line
the header is not working

https://nitro-docs-git-remove-rinkarby-offchain-labs.vercel.app/useful-addresses#token-bridge

File: arbitrum-docs/useful-addresses.mdx [hunk 7]
```diff
@@ -79,8 +79,8 @@ These are the addresses of some other gateways that have been deployed and integ
 ### Misc
```
Comment (fionnachan) [inline arbitrum-docs/useful-addresses.mdx:79]: same as above, need a line break before this line

File: arbitrum-docs/useful-addresses.mdx (hunk 3 above)
Comment (DZGoldman) [reply arbitrum-docs/useful-addresses.mdx:47]: good catch

File: arbitrum-docs/useful-addresses.mdx [hunk 8]
```diff
@@ -33,30 +33,31 @@ _Users should only interact with the token bridge via dapp interfaces like https
...
+| L2 Weth Gateway       | [0x6c411aD3E74De3E7Bd422b94A27770f5B86C623B](https://arbiscan.io/address/0x6c411aD3E74De3E7Bd422b94A27770f5B86C623B)  | [0x7626841cB6113412F9c88D3ADC720C9FAC88D9eD](https://nova.arbiscan.io/address/0x7626841cB6113412F9c88D3ADC720C9FAC88D9eD) | [0xf9F2e89c8347BD96742Cc07095dee490e64301d6](https://goerli.arbiscan.io/address/0xf9F2e89c8347BD96742Cc07095dee490e64301d6)  |
+| L1 Weth               | [0xc02aaa39b223fe8d0a0e5c4f27ead9083c756cc2](https://etherscan.io/address/0xc02aaa39b223fe8d0a0e5c4f27ead9083c756cc2) | [0xC02aaA39b223FE8D0A0e5C4F27eAD9083C756Cc2](https://etherscan.io/address/0xC02aaA39b223FE8D0A0e5C4F27eAD9083C756Cc2)     | [0xB4FBF271143F4FBf7B91A5ded31805e42b2208d6](https://goerli.etherscan.io/address/0xB4FBF271143F4FBf7B91A5ded31805e42b2208d6) |
+| L2 Weth               | [0x82aF49447D8a07e3bd95BD0d56f35241523fBab1](https://arbiscan.io/address/0x82aF49447D8a07e3bd95BD0d56f35241523fBab1)  | [0x722E8BdD2ce80A4422E880164f2079488e115365](https://nova.arbiscan.io/address/0x722E8BdD2ce80A4422E880164f2079488e115365) | [0xe39Ab88f8A4777030A534146A9Ca3B52bd5D43A3](https://goerli.etherscan.io/address/0xe39Ab88f8A4777030A534146A9Ca3B52bd5D43A3) |
```
Comment (fionnachan) [inline arbitrum-docs/useful-addresses.mdx:47]: Should be arbiscan as it's the nitro goerli column

Comment (DZGoldman) [inline arbitrum-docs/useful-addresses.mdx:47]:
```suggestion
| L2 Weth               | [0x82aF49447D8a07e3bd95BD0d56f35241523fBab1](https://arbiscan.io/address/0x82aF49447D8a07e3bd95BD0d56f35241523fBab1)  | [0x722E8BdD2ce80A4422E880164f2079488e115365](https://nova.arbiscan.io/address/0x722E8BdD2ce80A4422E880164f2079488e115365) | [0xe39Ab88f8A4777030A534146A9Ca3B52bd5D43A3](https://goerli.arbiscan.io/address/0xe39Ab88f8A4777030A534146A9Ca3B52bd5D43A3) |
```

Comment (DZGoldman) [reply arbitrum-docs/useful-addresses.mdx]: derp

**Review by fionnachan — APPROVED** [review]
LGTM

**Pull request merged**